plaidbench/plaidbench.py

```diff
@@ -180,7 +186,10 @@ def main():
             argv.append('--refresh-onnx-data is only meaningful with --onnx')
         if args.fix_learn_phase:
             argv.append('--fix-learn-phase')
-        if args.plaid or (not args.no_plaid and not args.caffe2):
+        if args.plaid_edsl:
+            argv.append('--plaid-edsl')
+            os.environ["KERAS_BACKEND"] = "plaidml2.bridge.keras.__init__"
```

Comment:
BTW, I don't think the `__init__` is required

Reply:
You're right, it's not